rfc/rfc006-distributed-registry.md

```diff
@@ -279,7 +278,10 @@ The keys of that controller can be kept offline.
 A stolen key can alter the DID document in such a way that the attacker can get full control with a new key and can exclude the previous owner from making changes.
 Appropriate measures MUST be taken to keep authentication keys secure. 
 
-When control over a DID document has been lost, the DID subject will have to have all Verifiable Credentials revoked. 
+The consequences of a theft can be mitigated by making a chain of controllers where each DID Document can only be altered by its controller.
+The root must be in control of its own DID Document, the key for this root should be stored offline (like a vault or offline machine).
```

Comment:
"The root" might be confusion, explain a little bit, e.g.;

"The root" -> "The root DID document (which doesn't delegate control to a controller)"